testsuite/tests/ui/search/test_account_search.py

```diff
@@ -94,3 +93,24 @@ def test_search_non_existing_value(request, login, navigator, custom_account):
     accounts.search("non-existing-value")
 
     assert next(accounts.table.rows())[0].text == "No results."
+
+
+# pylint: disable=unused-argument
+@pytest.mark.xfail
+@pytest.mark.issue("https://issues.redhat.com/browse/THREESCALE-8468")
+def test_search_short_keyword(login, navigator, custom_account, request):
+    """
+    Preparation:
+        - Create custom account with short keyword (less than 3 characters)
+    Test if:
+        - you search account by specific org_name it will return the correct one
+    """
+    org_name = blame(request, "org-1")
+    params = dict(org_name=org_name, username="username", email="email@anything.invalid",
```

Comment:
Please change the username to something else than "username" as it is already used in this test module.